packages/plugin-core/src/components/views/PreviewPanel.ts

```diff
@@ -137,7 +138,7 @@ export class PreviewPanel implements PreviewProxy, vscode.Disposable {
           this._onTextChanged = undefined;
         }
 
-        this._panel = undefined;
+        this.dispose();
```

Comment:
@kevinslin not super confident about this change. Maybe you can check if this does not break existing behavior.

Reply:
this shouldn't be necessary since this is invoked after `panel.dispose` is called

Reply:
ok so you are pointing to `this._panel = undefined;` being called in  `dispose()` meaning we are confident about that change and can leave it like it is.

Reply:
if by leaving as is you mean reverting this change (not adding `this.dispose`), than yes. `this.dispose` invokes `this._panel.onDidDispose` 
adding @jonathanyeung in case he has more to add here

Reply:
@jonathanyeung the intention with this commit was to go after behavoir 1.

[`d13cef8` (#3293)](https://github.com/dendronhq/dendron/pull/3293/commits/d13cef8231ee2c9e8bc39ce2a597a31ed38b54dc)

@kevinslin after some [delayering/untangling](https://twitter.com/Malcolm_Ocean/status/1363299005443108864) my confusion was about not seing `dispose` and `onDidDispose` being two different entry points. obvious now :)